custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py

```diff
@@ -191,7 +194,8 @@ def _run_java_server_entry_point(self):
                 java_cp,
                 JavaPredictor.JAVA_COMPONENT_ENTRY_POINT_CLASS,
                 "--class-name",
-                JavaPredictor.java_class_by_ext[self.model_artifact_extension],
+                self._custom_predictor_class
```

Comment:
I suggest to add a log message (info) that a custom predictor class was given and being used.

Reply:
Done